### PR TITLE
[QPPA-1469] Reformat and rewrite text for performance rate algorithms

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
[QPPA-1469] Reformat and rewrite text for performance rate algorithms

https://jira.cms.gov/browse/qppa-1469

Test plan: Tested working locally, see screenshot:
screen shot 2017-12-20 at 12 18 19 pm

### CODE REVIEW

File: src/components/api-reference/schemas/measurements.js [hunk 1]
```diff
@@ -54,7 +54,18 @@ const STRATA_FIELDS = [
...
+    name: 'performanceRate',
+    value: 'float',
+    description: 'Performance rates are stored and represented as percentages with values from 0.00 to 100.00. ' +
+      'The performance rate for a multiple performance rate measurement is calculated using the overallAlgorithm of the corresponding measure. ' +
+      'Note, performance rate calculations exclude any "overall" strata. ' +
+      'Also note, for the <b>simpleAverage</b> algorithm, each stratum\'s performance rate is calculated as <samp>performanceMet / (performanceMet + performanceNotMet - exclusions)</samp>. ' +
+      'Currently, a measure\'s overallAlgorithm may be one of the following:<ul>' +
+      '<li><b>simpleAverage:</b> <samp>(sum strata\'s performance rates) / (number of strata) * 100</samp>,</li>' +
+      '<li><b>weightedAverage:</b> <samp>((sum strata\'s performanceMet) / (sum strata\'s performanceMet and performanceNotMet)) * 100</samp>, or,</li>' +
+      '<li><b>sumNumerators:</b> <samp>sum strata\'s performanceMet</samp>.</li></ul>'
```
Comment (marimiyachi) [inline src/components/api-reference/schemas/measurements.js]: There is also the overallStratumOnly algorithm which calculates only based on the "overall" stratum.

File: src/components/api-reference/schemas/measurements.js [hunk 2]
```diff
@@ -54,7 +54,18 @@ const STRATA_FIELDS = [
...
-  {name: 'performanceRate', value: 'float', description: 'The performance rate for a multiple performance rate measurement is calculated based on the overallAlgorithm of the corresponding measure. Currently, a measure\'s overallAlgorithm can be a <b>simpleAverage</b>: ((sum of performanceMet from all of the non-overall strata) / (sum of performanceMet and performanceNotMet from all of the non-overall strata)) / (number of non-overall strata), <b>weightedAverage</b>: ((sum of performanceMet from all of the non-overall strata) / (sum of performanceMet and performanceNotMet from all of the non-overall strata)) * 100, or <b>sumNumerators</b>: the sum of the performanceMet property from each non-overall stratum.'}
+  {
+    name: 'performanceRate',
+    value: 'float',
+    description: 'Performance rates are stored and represented as percentages with values from 0.00 to 100.00. ' +
+      'The performance rate for a multiple performance rate measurement is calculated using the overallAlgorithm of the corresponding measure. ' +
+      'Note, performance rate calculations exclude any "overall" strata. ' +
```
Comment (marimiyachi) [inline src/components/api-reference/schemas/measurements.js]: super nit: only one "overall" so it's a stratum.

Comment (abarciauskas-bgse) [reply src/components/api-reference/schemas/measurements.js]: I keep trying to think about this by replacing stratum(a) with apple(s), and you would say `performance rate calculations exclude any "overall" apples`

File: src/components/api-reference/schemas/measurements.js (hunk 1 above)
Comment (abarciauskas-bgse) [reply src/components/api-reference/schemas/measurements.js]: Good call, added in https://github.com/CMSgov/qpp-submissions-docs/pull/83/commits/eacc7376cdbf6a6c03164795c0d81765501a7384

**Pull request closed without merging**